GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationViewModel.swift

```diff
@@ -58,7 +58,7 @@ extension SecureConversations.ConfirmationViewModel {
         return Header.Props(
             title: style.headerTitle,
             effect: .none,
-            endButton: .init(style: style.header.endButton),
+            endButton: .init(style: style.header.endButton, accessibilityIdentifier: "header_end_button"),
```

Comment:
Do we have a file with constants? This is used so much that it would be cool to consolidate it in one place.

Reply:
we don't, however, we can think about it to combine all accessibility identifiers together and use them. Would you like me to do it in this PR?

Reply:
Maybe a ticket can be created for cooldown